uglify-mangle-excludes.ts

```diff
@@ -10,5 +10,9 @@ export default [
     // Android    
     "DropDownAdapter",
     "DropDownItemSelectedListener",
-    "DropDownTouchListener"
+    "DropDownTouchListener",
+
+    // Angular
```

Comment:
Also do we need these angular specific items to be excluded from mangling? From what I see none of the similar things in the `nativescript-angular` are excluded.